.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -59,11 +59,13 @@ body:
     attributes:
       label: Environment
       description: Run the `npx expo-env-info` command and paste its output in the field below.
+      render: text
```

Hello @oblador, thanks for catching this!

```suggestion
      render: shell
```

According to GitHub docs there is also a `shell` value, which might work here better:
* https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms

---

Hi, I know about the `shell` one and use it in my READMEs etc, but to me it doesn't fit the bill as shell is for shell code, not shell _output_. 

Compare plain text:

```text
if this were text it should all be same color
```

With shell:

```shell
if this were text it should all be same color
```